code/extensions/GoogleSitemapExtension.php

```diff
@@ -33,7 +33,8 @@ public function canIncludeInGoogleSitemap()
             $can = $this->owner->getGooglePriority();
         }
 
-        $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can);
+        // Allow override. In this case, since we don't know if this should be an "and" vs. "or" override, we'll use the LAST result (if any).
+        if ($override = $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can)) $can = array_pop($override);
```

Comment:
No need to ignore nulls, I believe those are never pushed into the array returned by `->extend()` (called by `->invokeWithExtensions()`), so we're good there. I think my problem was that I have a single site spread out across multiple domains and this plugin thinks it's being helpful by automatically assuming that it shouldn't be included. If there are multiple extensions tinkering with results, we have ourselves a little issue (then) -- of course at fault of the developer. In this case the question is how to handle that at a lower extension level.